src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+$days: ("mon":0, "tue":1, "wed":2, "thu":3, "fri":4, "sat":5, "sun":6);
+$cell_width:100px;
+$gap:1px;
+$month_length:(28,29,30,31);
+@mixin calendar_block_cell {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
Looks strang, do we really need this gap?

Reply:
yes, we have it in readme.md file '- Use flex with `1px` gap and limit its width to exactly 7 columns + `10px` paddings'

Reply:
yes, we have it in readme.md file '- Use flex with `1px` gap and limit its width to exactly 7 columns + `10px` paddings'